tests/test_slogpy_console.py

```diff
@@ -0,0 +1,40 @@
+"""The purpose of the code is to give debug method a unit test to make sure that they are working properly"""
```

Comment:
You appear to be testing both info() and debug().  You probably have some aspirations for the code that will live here eventually (based on the name you chose for the file).  So, it would be a good idea to change the docstring to be more accurate, since it likely reflects neither what the code does now, nor you final intentions.

Reply:
got it